public/js/components/App.js

```diff
@@ -29,7 +29,7 @@ const App = React.createClass({
 
   getInitialState() {
     return {
-      searchOn: false
+      searchOn: true
```

Comment:
Yeah, i have mixed feelings about opening it by default. I added that to the list because it was discussed when we released autocomplete:

https://github.com/devtools-html/debugger.html/issues/338

I personally would be happy to keep it closed by default.